Fabric.Authorization.AccessControl/src/app/services/browser-requirements.service.ts

```diff
@@ -0,0 +1,26 @@
+import { Injectable } from '@angular/core';
+import { Observable } from 'rxjs';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class BrowserRequirementsService {
+
+  constructor() { }
+
+  cookiesEnabled(): boolean {
+    const userAgent = window.navigator.userAgent;
+    if (!(userAgent.indexOf('MSIE ') > -1 || userAgent.indexOf('Trident') > -1) &&
+        typeof navigator.cookieEnabled !== 'undefined' && navigator.cookieEnabled) {
+      return true;
+    }
+    if (!document.cookie || document.cookie === '') {
```

Comment:
looks like this tests both chrome based browsers and firefox, is that right?

Reply:
It tests appropriately for Chrome, Firefox and IE.